### DISCUSSION TITLE
Fluent UI and Tailwind CSS

### DISCUSSION THREAD

**Discussion by dinogit** [microsoft/fluentui#31682] [Ideas]
Hi all,

I have just started working with Fluent UI after 2 years of building apps based on Tailwind CSS design systems, and for me just adding CSS classes instead of building Griffel objects is a more intuitive, faster, and easier way to build components. 

I have tried adding Tailwind classes to components, using `mergeClasses`, but it does not have any effect. 

If I want to build Fluent UI with Tailwind CSS, would building my own FluentProvider be a step in the right direction?

**Comment by jay-itl** [comment]
I have have done is using `@apply` in the module css for customization
